### ISSUE TITLE
shuffle plugin

### ISSUE THREAD

**Issue by argyleink** [GoogleChromeLabs/ProjectVisBug#335]
be really neat and super easy to make a /command that shuffled the children of the selected elements.

**imagine clicking a grid container then typing `/shuffle` and letting a random function try out a new arrangement for you.**

**Comment by argyleink** [comment]
https://github.com/GoogleChromeLabs/ProjectVisBug/wiki/Plugins

**Comment by fxnoob** [comment]
` export const commands = []

export default async function(selected) {
  // selected is optional 
  // selected is a nodeList containing the elements currently selected with VisBug
} `
//selected contains command string instead of a nodeList containing the elements currently selected with VisBug

**Comment by fxnoob** [comment]
here is my implementation 
https://github.com/fxnoob/ProjectVisBug/commit/8360f2eeb4240c6a0f85bfcd7bfe055521b398f5

but getting command string in default function arg instead of highlighted node.
Please help!

**Comment by argyleink** [comment]
very sorry for the misleading docs and incomplete api, you are correct, visbug was wrong. 

that being said, i've fixed it, updated the docs, and it should only require a minor change to your code:

```js
export default async function({selected}) {
  // destructure selected from the incoming params
}
```

this has been merged into master, pull latest to be unblocked! lemme know how it goes, thanks!
